config/routes.rb

```diff
@@ -1,2 +1,15 @@
 Rails.application.routes.draw do
+  root 'sessions#new'
+
+  get  'signup', to: 'users#new'
+  post 'signup', to: 'users#create', as: 'users'
+
+  get  'login', to: 'sessions#new'
+  post 'login', to: 'sessions#create'
```

Comment:
If use signup then write in the same way. Write `signin` or `sign_in` instead of `login`